components/Map/MapDetail.tsx

```diff
@@ -1,25 +1,43 @@
 import React from 'react';
 import { View, HeaderText } from '../Themed';
-import { StyleSheet } from 'react-native';
+import { StyleSheet, ScrollView } from 'react-native';
+import { BottomSheetScrollView, BottomSheetView } from "@gorhom/bottom-sheet";
+
 
 interface MapDetailProps {
   title: string;
   description?: string;
 }
 
 const MapDetail: React.FC<MapDetailProps> = (props: MapDetailProps) => {
-  return (
-    <View style={styles.container}>
-      <HeaderText style={styles.title}>{props.title}</HeaderText>
-      {props.description ? <HeaderText>{props.description}</HeaderText> : null}
-    </View>
-  );
+  if (props.description)
+  {
+    return (<>
+        <HeaderText style={styles.title}>{props.title}</HeaderText>
+        <BottomSheetScrollView
+            stickyHeaderIndices={[0]}>
+          <HeaderText></HeaderText>
```

Comment:
Waarom de lege Header? If needed, zou het dan eerder als `<HeaderText />` schrijven

Reply:
anders werkt het niet 🤷 Please probeer het te fixen haha
Er is iets mis met die HeaderIndices als ik dit niet toevoeg dan is het kapot. 
Als ik de headertext met de title hierin zet en link als sticky header index, dan is het ook kapot. 
Op deze manier werkt het, maar geen idee waarom.
🙃

Reply:
Denk dat ik het gefixt heb, works on my machine met een giga lorem ipsum